### DISCUSSION TITLE
トラックボールの速度調整はどうすればよいでしょうか？

### DISCUSSION THREAD

**Discussion by mura-** [Yowkees/keyball#101] [Q&A] [answered]
defaultだとLayer3の右のほうあたりを入力すると速度が変わるように見受けられました。
<img width="952" alt="image" src="https://user-images.githubusercontent.com/1659899/159127569-e48458d1-525c-428b-97aa-94669b11f590.png">

ここらへんの仕様がまとまったドキュメントなどはありますでしょうか？
初心者＆見逃していたらすいません。

**Comment by Yowkees** [comment, marked as answer]
Keyballシリーズのファームウエアでは、カーソル速度（CPI）とスクロール速度を個別に調整可能です。
Anyキーと表示せれてしまっているキーの正式名称はdefaultのキーマップと照合していただければと思います。
https://github.com/Yowkees/keyball/blob/main/qmk_firmware/keyboards/keyball/keyball61/keymaps/default/keymap.c

各キーの動作はKeyball61の紹介ページの下の方に載っています。
https://github.com/Yowkees/keyball/tree/main/qmk_firmware/keyboards/keyball/keyball61

ご不明点ございましたらご連絡ください。

**Reply by mura-** [reply]
なるほど、理解いたしました。ありがとうございます。
